armi/tests/tutorials/c5g7-blueprints.yaml

```diff
@@ -65,9 +65,8 @@ custom isotopics:
         input format: number densities
         H: 6.70e-2
         O: 3.35E-2
-        B: 2.87E-5
-        U235: 6.0e-8
-        U238: 6.0e-9
```

Comment:
Although the reference states 1.0e-8 atom/b-cm, does the comment of ~90% enrichment still apply? If so, can we add U-238 at this fraction or remove the comment?

Reply:
>  I will check it out in more depth.

Can't wait!